pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -343,16 +343,22 @@ spec:
     {{- jsonpath .JsonResult "{.metadata.name}" | trim | saveAs "cvolcreateputvolume.objectName" .TaskResult | noop -}}
   task: |
     {{- $replicaCount := .Config.ReplicaCount.value | int64 -}}
+    {{- $isClone := .Volume.isCloneEnable | default "false" -}}
```

Comment:
it seems - `isCloneEnable` tells us that the volume is clone. I understand that isCloneEnable is an existing one.. but, the name doesn't sound correct. It should have been just 'isClone'.

Reply:
Right.. agree ! Can we take this change as part of other PR's ?.